docs/src/static-pages/website-contribution-guide.md

```diff
@@ -22,7 +22,106 @@ To make changes to the website's content you'll need a GitHub account. If you do
 
 Pages on this website are written using [Markdown](https://www.markdownguide.org/getting-started/), a markup language for writing documents. It's very simple and you don't need any knowledge of software engineering to use it.
 
-## Steps
+## Get started
+
+The instructions are different, depending on whether you want to add a new page or change an existing page.
+
+* [Add a new page](#add-a-new-page)
+* [Change an existing page](#change-an-existing-page)
+
+## Add a new page
+
+### 1. Confirm that it's an editable section
+
+Not all parts of the website can be updated through this process. Currently, the _Using Backpack_ and _Guidelines_ sections can be changed using these instructions.
+
+> We're working on making more sections compatible with this process.
+
+### 2. Go to GitHub
+
+Go to the [`static-pages` folder on GitHub](https://github.com/Skyscanner/backpack-docs/tree/master/docs/src/static-pages), which is where all the markdown files for these pages are kept. This is where you'll add your new file that will be turned into a page on the website.
+
+Press 'Add file', then in the drop down choose 'Create new file'.
+
+![The static pages folder on GitHub](../static/website-contribution-guide/static-pages-github.png)
+
+### 3. Add your content
+
+You'll be taken to a blank page where you can add your file's content.
+
+![GitHub's "new file" page](../static/website-contribution-guide/blank-new-file.png)
+
+
+#### 3.1 Choose a file name
+
+Enter a file name for your page in the "Name your file..." text box. File names must be in lower case and can contain letters, numbers and hyphens. Spaces aren't allowed.
+
+> Your filename must end with `.md`, so that GitHub identifies it as a markdown file.
+
+If you're unsure, go back to the previous step and use the existing files for inspiration.
+
+#### 3.2 Add the page information
+
+Your file must begin with a small block of text that includes some details about the page. Copy one [from an existing page](https://raw.githubusercontent.com/Skyscanner/backpack-docs/master/docs/src/static-pages/website-contribution-guide.md) and paste it into your new page. `category` should either be `using` or `guidelines`.
```

Comment:
I did originally have a thing that users could just copy here, like this:

```
---
title: Your title here
subtitle: One sentence describing your page
category: Put 'using' or 'guidelines' here
---
```

But there's something about putting `---` in the Markdown that blows up the renderer. I tried all sorts of escaping it but couldn't get it to work, so I just removed it and changed it to a link.